### PR TITLE
ENH: add code to check that all mriqc reports were generated

### PR DESCRIPTION
**Pull request by celprov**
enh: add code to check that all mriqc reports were generated
fix: fix secrets about sherlock path

### CODE REVIEW

File: code/qc/check_all_reports_generated.py
Comment (oesteban) [inline code/qc/check_all_reports_generated.py:1]: Please add shebang, license banner and run through ruff / black (seems like ruff is better).

File: code/qc/check_all_reports_generated.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+missing_reports = {}
+for session in sessions:
+
+    # Initialize a list to store missing reports for this session
+    missing_reports[session] = []
+
+    # Iterate over each pattern
+    for pattern in patterns:
+        # Construct the regex pattern
+        regex_pattern = f"sub-001_ses-{session}_{pattern}.html"
```
Comment (oesteban) [inline code/qc/check_all_reports_generated.py]: Not really a regex:
```suggestion
        glob_pattern = f"sub-001_ses-{session}_{pattern}.html"
```

File: code/qc/check_all_reports_generated.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        # Use glob to check if any report files matching the pattern exist
+        report_files = glob.glob(os.path.join(derivatives_dir, regex_pattern))
```
Comment (oesteban) [inline code/qc/check_all_reports_generated.py]:
```suggestion
        report_files = glob.glob(os.path.join(derivatives_dir, glob_pattern))
```

File: docs/data-management/mriqc.md [hunk 3]
```diff
@@ -39,6 +39,12 @@
         │   ├── sub-001_ses-001_task-qct_dir-AP_bold.html
         │   ├── sub-001_ses-001_task-rest_dir-AP_bold.html
     ```
+    - [ ] Adapt the following code to your setup to check this automatically. Adapt it to your setup.
+    ```shell
+{% filter indent(width=4) %}
+{% include 'code/qc/check_all_reports_generated.py' %}
+{% endfilter %}
+    ```
```
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
        ```shell
{% filter indent(width=8) %}
{% include 'code/qc/check_all_reports_generated.py' %}
{% endfilter %}
        ```
```

File: docs/data-management/post-session.md [hunk 4]
```diff
@@ -491,7 +491,7 @@ As new sessions are collected, the corresponding BIDS structures MUST be saved w
 
     ``` shell
 
-    docker run -ti --rm -v {{ secrets.data.path_data_sherlock | default('/path/to/data/') }}:/data:ro bids/validator /data
+    docker run -ti --rm -v {{ secrets.celine_paths.data_sherlock | default('/path/to/data/') }}/hcph/:/data:ro bids/validator /data
```
Comment (oesteban) [inline docs/data-management/post-session.md]: don't forget to add this config :)

**Review by oesteban — APPROVED** [review]

File: code/qc/check_all_reports_generated.py
Comment (celprov) [reply code/qc/check_all_reports_generated.py:1]: Sorry you have to repeat running ruff everytime ^^'

**Pull request merged**